template.yaml

```diff
@@ -89,9 +76,9 @@ Resources:
     Properties:
       Environment:
         Variables:
-          AllowOrigin: !Ref  CorsOrigin
+          ALLOWORIGIN: "'*'"
       Handler: no.unit.nva.cristin.institutions.FetchCristinUnit::handleRequest
-      Runtime: java8
+      Runtime: java11
       MemorySize: 512
       Events:
         NvaCristinFetchUnitEvent:
```

Comment:
I think you are missing a RequestParameter for {id}

Reply:
Added